Dockerfile

```
@@ -72,6 +72,11 @@ RUN \
     node_modules \
     dist \
     /app/immich/server && \
+  echo "**** copy server immich-healthcheck ****" && \
+  mkdir -p \
+    /app/immich/server/bin && \
+  cp /tmp/immich/server/bin/immich-healthcheck \
```

just like everywhere else
```suggestion
  cp -a 
      /tmp/immich/server/bin/immich-healthcheck \
```